### PR TITLE
fix: Fix StackScript test by using latest images available

### PR DESCRIPTION
**Pull request by jdamore-linode**
## Description 📝
Our StackScript test broke because one of the Images the test selects, `Ubuntu 22.10`, became deprecated and tripped up a Cypress selector.

This quickly fixes the test by updating the images used by the tests to their latest versions.

There's potential to fix this permanently by making an API request to fetch the same data that's shown in Cloud, but this happens so infrequently that today I opted to fix it quickly but plan to revisit this in the future.

## How to test 🧪
We can rely on the automated tests for this, but reviewers can use the following command to run the affected test locally:

```bash
yarn cy:run -s "cypress/e2e/core/stackscripts/create-stackscripts.spec.ts"
```

### CODE REVIEW

**Review by bnussman-akamai — APPROVED** [review]

**Review by mjac0bs — COMMENTED** [review]
I tried running this locally a few times and was never able to get the second test to pass; it seems to consistently time out. In the video, the test just gets to the stackscript create page and never fills out the form. 

Screenshot 2023-07-20 at 1 16 51 PM

**Comment by jdamore-linode** [comment]
@mjac0bs Yeah, that's the test timing out while creating an image to use for the test -- not really related to this fix and there isn't much I can do for that besides increase the timeout (which I'll do as soon as GitHub starts working again..).

One possibility is that your test account has internal regions enabled, and the test is using one which doesn't have Machine Images working yet. You can eliminate that possibility by forcing the test to run against a specific region like `us-east`:

```bash
CY_TEST_REGION=us-east yarn cy:run -s "cypress/e2e/core/stackscripts/create-stackscripts.spec.ts"
```

File: packages/manager/cypress/e2e/core/stackscripts/create-stackscripts.spec.ts [hunk 1]
```diff
@@ -258,7 +264,7 @@ describe('Create stackscripts', () => {
    * - Confirms that private images can be selected when deploying with "Any/All" StackScripts.
    * - Confirms that a Linode can be deployed using StackScript with private image.
    */
-  it('creates a StackScript with Any/All target image', () => {
+  it.only('creates a StackScript with Any/All target image', () => {
```
Comment (bnussman-akamai) [inline packages/manager/cypress/e2e/core/stackscripts/create-stackscripts.spec.ts:267]: Do we still want the `only`?

**Review by bnussman-akamai — APPROVED** [review]

**Review by cpathipa — APPROVED** [review]

File: packages/manager/cypress/e2e/core/stackscripts/create-stackscripts.spec.ts (hunk 1 above)
Comment (jdamore-linode) [reply packages/manager/cypress/e2e/core/stackscripts/create-stackscripts.spec.ts:267]: Shoot lol, thanks for catching this

**Review by coliu-akamai — APPROVED** [review]
commented out the .only on my local to see if they both passed, looks good!

**Review by carrillo-erik — APPROVED** [review]

**Comment by carrillo-erik** [comment]
Both passing  ✅

Screenshot 2023-07-21 at 11 52 05 AM

**Pull request merged**